test/widgets/lightbox_test.dart

```diff
@@ -236,14 +236,19 @@ void main() {
       debugNetworkImageHttpClientProvider = null;
     });
 
-    testWidgets('app bar shows sender name and date', (tester) async {
+    testWidgets('app bar shows sender avatar, name and date', (tester) async {
       prepareBoringImageHttpClient();
       final timestamp = DateTime.parse("2024-07-23 23:12:24").millisecondsSinceEpoch ~/ 1000;
       final message = eg.streamMessage(sender: eg.otherUser, timestamp: timestamp);
       await setupPage(tester, message: message, thumbnailUrl: null);
 
-      // We're looking for a RichText, in the app bar, with both the
-      // sender's name and the timestamp.
```

Comment:
nit: the comment was specifically for explaining what `RichText` was used for. Let's leave it as-is

Reply:
The comment belongs to the part started with `final labelTextWidget`, so it should come before that and replace `// Check sender name and timestamp`.